vocabularies/voc4cat/0007007.ttl

```diff
@@ -5,7 +5,13 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007007>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007007"^^xsd:token ;
-    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000184> ;
+    skos:altLabel
+        "stability experiment"@en ,
+        "stability study"@en ,
```

Comment:
A study typically consists of multiple experiments. So can you remove "stability study" from the alternative labels?